x/auth/account.go

```diff
@@ -35,6 +36,9 @@ type Account interface {
 	// Calculates the amount of coins that can be sent to other accounts given
 	// the current time.
 	SpendableCoins(blockTime time.Time) sdk.Coins
+
+	// Ensure that account implements stringer
+	String() string
```

Comment:
Note addition of `String()` to account interface (i.e. each different kind of account must have a human readable representation)